image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -259,10 +259,10 @@ defaults:
                 operator: DoesNotExist
       resources:
         requests:
-          memory: "4Gi"
+          memory: "500Mi"
```

Comment:
LGTM. Do we need to set to 1G just in case?

Reply:
Seems like a single instance is only maxing at most 200Mi, so I think 500Mi is good

Reply:
Also note we set the Go runtime soft memory limit to 90% of the request, so it'll be 450Mi. Again it's a soft limit, but if set too low may cause GC thrashing. Again based on the graphs, seems like they max at 200Mi, so this should be sufficient

Reply:
If the instance is maxing at 200Mi, shouldn't 200Mi be the `limit` not the `request`?

Reply:
This looks about right. Hovering at 750Mi
<img width="1666" alt="Screenshot 2024-07-03 at 10 19 29 AM" src="https://github.com/stackrox/stackrox/assets/38968/57123ab1-3b4e-4283-80c9-33ee9f41e2cb">

Reply:
> If the instance is maxing at 200Mi, shouldn't 200Mi be the limit not the request?

We are being conservative; We don't know yet what can cause spikes in the Matcher's memory consumption at scale. And the improvements done in the vulnerability updater are localized to writing vulnerabilities. Keeping the limit high now so it won't unnecessarily OOM, while requiring what we believe is enough for a healthy Matcher.